src/PKSim.Core/Model/PopulationSimulation.cs

```diff
@@ -280,27 +279,37 @@ private DataColumn medianAggregateDataColumns(IReadOnlyList<DataColumn> columns)
          return  new DataColumn(column.Id, column.Dimension, column.BaseGrid)
          {
             Values = Enumerable.Range(0, column.Values.Count).Select(i => 
-                  IndexedValuesFromColumns(columns, i).Median()
+                  indexedValuesFromColumns(columns, i).Median()
                ).ToList()
          };
       }
 
-      private static IReadOnlyList<float> IndexedValuesFromColumns(IReadOnlyList<DataColumn> columns, int i)
+      private static IReadOnlyList<float> indexedValuesFromColumns(IReadOnlyList<DataColumn> columns, int i)
       {
          return columns.Select(x => x.Values[i]).ToList();
       }
 
       public override DataColumn PeripheralVenousBloodColumn(string compoundName)
       {
-         return medianAggregateDataColumns(drugColumnFor(CoreConstants.Organ.PERIPHERAL_VENOUS_BLOOD, CoreConstants.Observer.PLASMA_PERIPHERAL_VENOUS_BLOOD, CoreConstants.Observer.PLASMA_PERIPHERAL_VENOUS_BLOOD, compoundName));
+         return medianAggregateDataColumns(PeripheralVenousBloodColumns(compoundName));
+      }
+
+      public IReadOnlyList<DataColumn> PeripheralVenousBloodColumns(string compoundName)
```

Comment:
why is this public?

Reply:
GlobalPKAnalysisTask needs to get access to all the columns for PeripheralVenousBlood and VenousBlood